tests/testing/embeddings.py

```
@@ -0,0 +1,115 @@
+def _similarity(text: str, item_text: str, text_embedding: List[float], item_embedding: List[float]) -> float:
+    normalized_text = _normalized_text(text)
+    normalized_item_text = _normalized_text(item_text)
+
+    if normalized_text and (
+        normalized_text == normalized_item_text
+        or normalized_text in normalized_item_text
+        or normalized_item_text in normalized_text
+    ):
+        return 1.0
+
+    return sum(a * b for a, b in zip(text_embedding, item_embedding))
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)